src/EnergyPlus/UnitarySystem.hh

```diff
@@ -338,6 +400,7 @@ namespace UnitarySystems {
         bool m_SuppCompNotSetYet;
         bool m_OKToPrintSizing;
         Real64 m_SmallLoadTolerance;
+        bool m_setupOutputVars;
```

Comment:
variable used to hold off on call to setupOutputVariable until processing is complete. Not positively sure this will avoid duplicate calls but it fixed initial problems and matches what develop is doing (based on CI results).